src/java/org/apache/cassandra/service/accord/txn/TxnReferenceOperation.java

```diff
@@ -62,21 +68,22 @@ public class TxnReferenceOperation
     private static Map<Class<? extends Operation>, Kind> initOperationKindMap()
     {
         Map<Class<? extends Operation>, Kind> temp = new HashMap<>();
-        temp.put(Sets.Adder.class, Kind.SetAdder);
```

Comment:
i reorded so its easier to see something missing. `Lists.DiscarderByIndex` was missing!